### ISSUE TITLE
Make "Guzzle" the left click option on rock cake

### ISSUE THREAD

**Issue by ChiSox** [runelite/runelite#9488]
**Is your feature request related to a problem? Please describe.**
Right clicking and selecting guzzle is tedius in NMZ. Shift click is an option, still takes two hands so is not as ideal.

**Describe the solution you'd like**
Add "Guzzle" as an option in the menu swapper plug-in.

**Describe alternatives you've considered**
You can add it to shift-click configuration, but this is a perfect fit for menu swapper. It's something done by many players and shift-click takes two hands instead of one.

**Additional context**
Guzzling rock cakes for NMZ is used very often by tons of players. It is the recommended method to train on the wiki. This would be a huge QoL for everyone. It has advantages over shift-click in that it takes one hand, and making it native in menu swapper would give it wider use by everyone. I wasn't familiar with shift-click until today myself.

**Comment by Hydrox6** [comment]
Duplicate of #1860
TL;DR: use Shift Click configuration to make it work on Shift click
https://github.com/runelite/runelite/wiki/Menu-Entry-Swapper#shift-click-configuration
